.ci_support/build_notebooks.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+# execute notebooks
+i=0;
+for notebook in $(ls notebooks/*.ipynb); do
```

Shellcheck

[error] Iterating over ls output is fragile. Use globs.

(SC2045)